Jenkinsfile_CNP

```diff
@@ -79,14 +79,16 @@ def uploadCoreCaseDataDefinitions(env) {
   dir("${WORKSPACE}/bin"){
     if("preview" == env){
       ccdDefinitionFilePath="${WORKSPACE}/build/ccd-release-config/ccd-fpl-preview-${CHANGE_ID}-toggle-on.xlsx"
+      sh """
+        eval \$(./variables/load-preview-environment-variables.sh ${CHANGE_ID})
+        ${WORKSPACE}/fpla-docker/bin/utils/ccd-import-definition.sh ${ccdDefinitionFilePath}
+      """
     } else {
-      ccdDefinitionFilePath="${WORKSPACE}/build/ccd-release-config/ccd-fpl-${env}.xlsx"
```

Comment:
was this supposed to have been removed?

Reply:
what we will do without you Lewis.
fixed